docs/ossm/kiali/README.md

```diff
@@ -0,0 +1,463 @@
+# Kiali
+
+Kiali can be used to view the data flow through your application once it has been added to the service mesh.
+
+## About Kiali
+
+Kiali provides observability into the service mesh running on OpenShift. Kiali helps you define, validate, and observe your Istio service mesh. It helps you to understand the structure of your service mesh by inferring the topology, and also provides information about the health of your service mesh.
+
+Kiali provides an interactive graph view of your namespace in real time that provides visibility into features like circuit breakers, request rates, latency, and even graphs of traffic flows. Kiali offers insights about components at different levels, from Applications to Services and Workloads, and can display the interactions with contextual information and charts on the selected graph node or edge. Kiali also provides the ability to validate your Istio configurations, such as gateways, destination rules, virtual services, mesh policies, and more. Kiali provides detailed metrics, and a basic Grafana integration is available for advanced queries. Distributed tracing is provided by integrating Tempo into the Kiali console.
```

Comment:
I hesitate to tell people Kiali provides "real time" views. It doesn't really. Even down to the fact that Prometheus scrape intervals means Kiali isn't truly showing "real time" data.

I guess it depends on your definition of "real time".